triggers/jenkins-ansible_controller_configs.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: jenkins-ansible_controller_configs.yml
+  hosts: p-u-ansible01.library.ucla.edu
+  connection: local
+  gather_facts: false
+
+  vars:
+    ansible_github_url: git@github.com:UCLALibrary/ansible_uclalib_configs.git
+    ansible_github_config_path: /etc/ansible
+    ansible_github_branch: master
+    ansible_user: ansible
+    ansible_user_ssh_key_path: /home/ansible/.ssh/id_rsa
+
+  tasks:
+    - name: "Clone/Update configs repo"
+      git:
+        repo: "{{ ansible_github_url }}"
+        dest: "{{ ansible_github_config_path }}"
+        version: "{{ ansible_github_branch }}"
+        key_file: "{{ ansible_user_ssh_key_path }}"
+      become_user: "{{ ansible_user }}"
```

Comment:
ansible-lint:
> [501] become_user requires become to work as expected
> triggers/jenkins-ansible_controller_configs.yml:15
> Task/Handler: Clone/Update configs repo

yamllint (I agree with this one, and GitHub highlights it too):
> 21:40     error    no new line character at the end of file  (new-line-at-end-of-file)

Reply:
Thanks @jhriv, forgot to lint check these. I've fixed it and removed my editor from moving the last empty line. Please review again.